storybook/stories/Button/IconButton.js

```diff
@@ -22,11 +22,11 @@ const styles = StyleSheet.create({
 
 const IconButton = props => {
 	const buttonSizes = {
-		xlarge: theme.buttonSizeXLarge,
-		large: theme.buttonSizeLarge,
-		medium: theme.buttonSizeMedium,
-		small: theme.buttonSizeSmall,
-		xsmall: theme.buttonSizeXSmall
+		xlarge: (theme.iconSizeXLarge * 2 - 4),
```

Comment:
Why are we modifying the styles here?

Reply:
I removed the original definition for buttonSize from the theme file as we have more than one type of button.  Instead, I scale the size of the button based on the size of the icon.  Not what you would prefer @luigiinred ?